src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpStep.java

```diff
@@ -61,6 +64,30 @@ protected boolean processCurrentFileAndExtractPdpFeedFromSAEFile(File currentFil
         }
         return success;
     }
+    
+    protected boolean isTextFile(File file) {
+        String fileName = file.getName();
+        return StringUtils.endsWith(fileName, ".txt");
+    }
+    
+    protected void debugConcurStandardAccountingExtractFile(ConcurStandardAccountingExtractFile saeFile) {
+        if (LOG.isDebugEnabled()) {
+            if (saeFile != null) {
+                LOG.debug("debugConcurStandardAccountingExtractFile, " + saeFile.getDebugInformation());
+                if (saeFile.getConcurStandardAccountingExtractDetailLines() != null) {
+                    LOG.debug("debugConcurStandardAccountingExtractFile, Number of line items: " + saeFile.getConcurStandardAccountingExtractDetailLines().size());
+                    for (ConcurStandardAccountingExtractDetailLine line : saeFile.getConcurStandardAccountingExtractDetailLines()) {
+                        LOG.debug("debugConcurStandardAccountingExtractFile, " + line.getDebugInformation());
+                    }
+                } else {
+                    LOG.debug("debugConcurStandardAccountingExtractFile, The getConcurStandardAccountingExtractDetailLines is null");
+                }
+                
+            } else {
+                LOG.debug("debugConcurStandardAccountingExtractFile, The SAE file is null");
+            }
+        }
+    }
 
     protected void moveFile(File currentFile, String subPath) {
         File dropDirectory = new File(directoryPath + subPath + ConcurConstants.FORWARD_SLASH);
```

Comment:
I realize this method was not modified with these changes but why is a time-date-stamp being added so that the file name is being changed? Our other accept-reject file movement operations do not change the file name that was given to us. Those routines just move the files out of the processing directory into either the accept or reject folder.